### DISCUSSION TITLE
Controlling PayloadLength

### DISCUSSION THREAD

**Discussion by scopedog** [microsoft/msquic#3224] [Q&A] [answered]
I need to make PayloadLength to pass to CxPlatEncrypt divisible by 256 or 128.
What is the best way to do that?
Thank you.

**Comment by nibanks** [comment, marked as answer]
We don't have a way to control that. Why do you need to do that?

**Reply by scopedog** [reply]
Thank you for your reply.
I have implemented a new encryption algorithm and the best payload size is 256 (or 128) * N.
